config/application.rb

```diff
@@ -91,5 +91,8 @@ class Application < Rails::Application
 
     # The time(in minutes) until the `load_min_user_count` will be used for calculating server load
     config.x.load_join_buffer_time = ENV.fetch('LOAD_JOIN_BUFFER_TIME', 15).to_i.minutes
+
+    # Recording feature will be disabled, if set to 'true'. Defaults to false.
+    config.x.recording_disabled = ENV.fetch('RECORDING_DISABLED', 'false')
```

Comment:
If you do the conversion to a boolean here, e.g.
```ruby
config.x.recording_disabled = ENV.fetch('RECORDING_DISABLED', 'false).casecmp?('true')
```

then in the other checks below, you can just do conditionals like

```ruby
if Rails.configuration.x.recording_disabled
```

without needing further type conversions.